Misc/NEWS.d/next/Core and Builtins/2020-05-19-19-39-49.bpo-40679.SVzz9p.rst

```diff
@@ -0,0 +1 @@
+Error messages about invalid arguments now refer to a function by its ``__qualname__``.
```

Comment:
Is it possible to be a little more specific about which error messages? If not, maybe you can just say,  "Certain error messages..." (since it's not all such messages). Also, it might be better to use `:term:qualified name` (with the right backticks) since that would provide a link to information about `__qualname__`.